src/build/index.ts

```diff
@@ -0,0 +1,71 @@
+import path from "node:path";
+import process from "node:process";
+import fs from "node:fs/promises";
+import { type RollupOptions, rollup } from "rollup";
+
+import { JspmError, exists } from "../utils";
+import type { Flags } from "../types";
+import { RollupImportmapPlugin } from "./rollup-importmap-plugin";
+
+export default async function build(entry: string, options: Flags) {
+  if (!entry && !options.buildConfig) {
+    throw new JspmError(`Please provide entry for the build`);
+  }
+
+  let buildConfig: RollupOptions;
+  let outputOptions: RollupOptions["output"];
+
+  if (entry) {
+    if (!options.buildOutput) {
+      throw new JspmError(`Build output is required when entry is provided`);
+    }
+
+    const entryPath = path.join(process.cwd(), entry);
+    if ((await exists(entryPath)) === false) {
+      throw new JspmError(`Entry file does not exist: ${entryPath}`);
+    }
+    buildConfig = {
+      input: entryPath,
+      plugins: [RollupImportmapPlugin(options)],
+    };
+
+    outputOptions = {
+      file: path.join(process.cwd(), options.buildOutput),
```

Comment:
Please let's use the `dir` option to support multiple entry points and dynamic import by default.